miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/AdMob.kt

```diff
@@ -15,5 +15,16 @@ internal fun isAdMobProvided(): Boolean =
         false
     }
 
+/** Check whether hostapp provides admob dependency. */
+@Suppress("EmptyCatchBlock", "SwallowedException")
+internal inline fun <T> whenHasH5AdsWebViewClient(callback: () -> T) {
+    try {
+        Class.forName("com.rakuten.tech.mobile.miniapp.ads.MiniAppH5AdsWebViewClient")
+        callback.invoke()
+    } catch (e: ClassNotFoundException) {
+        e.printStackTrace()
```

Comment:
Maybe we should log a message that tells the developer they are missing `admob-latest` dependency in this case?